src/components/GameChat.tsx

```diff
@@ -33,6 +33,9 @@ function GameChat({
     ]);
     setChatMessage('');
   };
+  const removeKeyBoard = () => {
+    console.log('phaser keyboard removed');
```

Comment:
Guessing longterm this removeKeyboard function will have some other purpose? Is the intention behind this to switch focus?